rcongui/src/components/PlayersHistory/PlayerTile/PlayerHeader.js

```diff
@@ -100,9 +102,8 @@ export const PlayerHeader = pure(({ classes, player }) => {
         }
         secondary={
           <Link
-            target="_blank"
             color="inherit"
-            href={`${process.env.REACT_APP_API_URL}player?steam_id_64=${player.get("steam_id_64")}`}
+            onClick={() => history.push(`/player/${player.get("steam_id_64")}`)}
```

Comment:
If you use the `Link` class from the router you don't need to push in the history manually.

Example here: https://github.com/MarechJ/hll_rcon_tool/blob/8cbf77d4733f2fa09830e70dce743d6c3a93dbfc/rcongui/src/components/Header/header.js#L193

Also I don't see you import Link, are you sure this is compiling?

Reply:
> Also I don't see you import Link, are you sure this is compiling?

Link was already there and it s imported line 5